### PR TITLE
Flux 0.7.1

### PR DESCRIPTION
**Pull request by sanderson**

### CODE REVIEW

**Comment by sanderson** [comment]
@aanthony1243 The changelog in this PR currently is just a place holder created from items in Flux's git history. If I'm missing items or if items should be removed, just let me know.

File: content/flux/v0.7/about_the_project/releasenotes-changelog.md [hunk 1]
```diff
@@ -6,6 +6,41 @@ menu:
     weight: 1
 ---
 
+## v0.7.1 [2018-12-06]
+
+_A technical preview of Flux packaged with InfluxDB v1.7.2._
+
+### Features
+- Create memory package with a memory allocator.
```
Comment (aanthony1243) [inline content/flux/v0.7/about_the_project/releasenotes-changelog.md]: not user facing

File: content/flux/v0.7/about_the_project/releasenotes-changelog.md [hunk 2]
```diff
@@ -6,6 +6,41 @@ menu:
...
+- Add support for string comparison operators.
+- Introduce the parser package and outward interfaces
```
Comment (aanthony1243) [inline content/flux/v0.7/about_the_project/releasenotes-changelog.md]: not user facing

File: content/flux/v0.7/about_the_project/releasenotes-changelog.md [hunk 3]
```diff
@@ -6,6 +6,41 @@ menu:
...
+- Initial work on a scanner using Ragel.
```
Comment (aanthony1243) [inline content/flux/v0.7/about_the_project/releasenotes-changelog.md]: not user facing

File: content/flux/v0.7/about_the_project/releasenotes-changelog.md [hunk 4]
```diff
@@ -6,6 +6,41 @@ menu:
...
+- Copy parser tests to the internal parser package
```
Comment (aanthony1243) [inline content/flux/v0.7/about_the_project/releasenotes-changelog.md]: not user facing

File: content/flux/v0.7/about_the_project/releasenotes-changelog.md [hunk 5]
```diff
@@ -6,6 +6,41 @@ menu:
...
+- Add array indexing.
+- Add scanned values & bytes to Statistics.
+- Add String method to Token type
```
Comment (aanthony1243) [inline content/flux/v0.7/about_the_project/releasenotes-changelog.md]: not user facing

File: content/flux/v0.7/about_the_project/releasenotes-changelog.md [hunk 6]
```diff
@@ -6,6 +6,41 @@ menu:
...
+- Update `gonum` and `tdigest` dependencies.
```
Comment (aanthony1243) [inline content/flux/v0.7/about_the_project/releasenotes-changelog.md]: not user facing

File: content/flux/v0.7/about_the_project/releasenotes-changelog.md [hunk 7]
```diff
@@ -6,6 +6,41 @@ menu:
...
+- Add more restrictions for function type unification.
```
Comment (aanthony1243) [inline content/flux/v0.7/about_the_project/releasenotes-changelog.md]: not user facing

**Pull request merged**